zb-classifier.js

```diff
@@ -1329,7 +1331,6 @@ class ZigbeeClassifier {
           label: 'Fan',
           type: 'string',
           enum: [],
```

Comment:
Do you not need `HVAC_FAN_MODE` here anymore?

Reply:
No - it's referenced internally in parseFanModeAttr. I was thinking I could genericize it a bit more, but never got that far.